palantirnet/st-bernards
# MSSB-94: Adding gulp

## JIRA Ticket(s)

- See ticket https://palantir.atlassian.net/browse/MSSB-94

## Description

- Added `gulpfile.js`
- Updated Bourbon and Neat libraries to a complete version to remove compilation errors
- Created / updated README with instructions

## Testing instructions

- `cd web/themes/custom/blrfurther_d8`
- `npm install`
- `gulp watch`
- Change something in a stylesheet (ex. _base.scss) and check that sass has been compiled and that it affects the local site.
- Run `gulp sass` and make sure it generated `.map` files that can be committed
